app/main/lib/text_similarity.py

```diff
@@ -175,6 +188,9 @@ def insert_model_into_response(hits, model_key):
             hit["_source"]["model"] = model_key
     return hits
 
+def return_sources(results):
+    return [dict(**r["_source"], **{"index": r["_index"], "score": r["_score"]}) for r in results]
```

Comment:
can you add a comment for what this does?  I think it is replacing the underscore version of the names?

Reply:
Yep Can do